src/gui/mzroll/peakdetectiondialog.cpp

```diff
@@ -157,6 +158,11 @@ PeakDetectionDialog::PeakDetectionDialog(QWidget *parent) :
 }
 
 
+void PeakDetectionDialog::onReset()
+{
+    emit resetSettings(pdSettings->getSettings().keys());
```

Comment:
@rish9511 There's another variable called odSettings for options dialog. Typo issue possibilities in the future. Can we change the variable names to something safer?

Reply:
@shubhra-agrawal 
can you suggest something better ? 
od = options dialog
pd = peaks dialog

Reply:
@rish9511 peakSettings and optionSettings works as well.